templates/README.md

```
@@ -132,6 +183,10 @@ Credit: https://github.com/ripta/dotfiles/tree/master/zsh-custom/plugins/kube/te
 
 #### Karpenter
 - `nodeclaim-drift.tmpl` - NodeClaim drift status and reason (Drifted condition)
+- `pods-nodepools.tmpl` - Pods joined to their node's NodePool (Go template; requires `get pods,nodes`)
+
+#### ArgoCD
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)